askbot/models/__init__.py

```diff
@@ -4348,12 +4348,12 @@ def record_spam_rejection(
     dispatch_uid='record_question_visit'
 )
 
-#set up a possibility for the users to follow others
-try:
-    import followit
-    followit.register(User)
-except ImportError:
-    pass
+# set up a possibility for the users to follow others
```

Comment:
I didn't want to remove it. I get the following traceback if I leave it in,

```python
Traceback (most recent call last):
  File "manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/pirave/work/poc/github/askbot-devel/askbot/models/__init__.py", line 4354, in <module>
    followit.register(User)
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/followit/__init__.py", line 127, in register
    from followit.compat import User
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/followit/compat.py", line 40, in <module>
    User = get_user_model()
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/contrib/auth/__init__.py", line 137, in get_user_model
    return django_apps.get_model(settings.AUTH_USER_MODEL)
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/apps/registry.py", line 199, in get_model
    self.check_models_ready()
  File "/home/pirave/work/poc/github/venv1.7/local/lib/python2.7/site-packages/django/apps/registry.py", line 131, in check_models_ready
    raise AppRegistryNotReady("Models aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.
```
After looking around I found [this](http://stackoverflow.com/questions/25680803/django-1-7-upgrade-error-appregistrynotready-models-arent-loaded-yet). It says 
*You try to reference a model at the import time. It is no longer possible in Django 1.7. Django 1.7 allows you to use your models only after all of the applications are loaded. You should move this call to the ready callback of AppConfig*

I am not sure how to incorporate this in, so I left a message in the pull request